pipelined/bevy_sprite2/src/sprite.rs

```diff
@@ -37,3 +40,26 @@ impl Sprite {
         }
     }
 }
+
+/// System that resizes sprites that have their resize mode set to automatic
+pub fn sprite_auto_resize_system(
```

Comment:
I think this is fine for now (and i know this is the approach we currently use), but its worth calling out that we can probably do better than polling every texture for every sprite every frame. But I'm fine eating the perf cost (~0.9 milliseconds per frame for 67,000 sprites in bevymark_pipelined or ~3 fps).